map/api_mark_container.cpp

```diff
@@ -0,0 +1,18 @@
+#include "map/api_mark_container.hpp"
```

Comment:
nit: it's better to group includes like there:
```c++
#include "map/api_mark_container.hpp"

#include "map/api_mark_point.hpp"
#include "map/framework.hpp"
```
because there is explicitly specified that this source contains implementation for api_mark_container.hpp.